src/main/java/com/github/dockerjava/api/model/Container.java

```diff
@@ -30,11 +34,17 @@
     @JsonProperty("Image")
     private String image;
 
+    /**
+     * @since since {@link RemoteApiVersion#VERSION_1_21}
+     */
+    @JsonProperty("ImageID")
+    private String imageId;
+
     @JsonProperty("Names")
     private String[] names;
 
     @JsonProperty("Ports")
-    public Port[] ports;
+    public ContainerPort[] ports;
```

Comment:
Classes that are "relative" to a parent class and can be used only in its context should be nested (`Container.Port`).